src/components/HighTable/HighTable.tsx

```diff
@@ -372,93 +382,95 @@ export default function HighTable({
 
   const ariaColCount = data.header.length + 1 // don't forget the selection column
   const ariaRowCount = data.numRows + 1 // don't forget the header row
-  return <ColumnWidthProvider localStorageKey={`${cacheKey}:column-widths`}>
```

Comment:
I recommend setting "hide whitespace" for the review

![Screenshot From 2025-04-08 22-26-36](https://github.com/user-attachments/assets/6f0d8e79-c43d-4acb-bebd-9a4b0f19264b)